### PR TITLE
LPS-197491

### PR DESCRIPTION
**Pull request by SelenaAungst**
https://liferay.atlassian.net/browse/LPS-197491

@rebsilva Finally resending a new PR for https://github.com/liferay-forms/liferay-portal/pull/2337 after figuring out the changes with the PEDS team. The only form field that still needs worked on is the Rich Text field, but we will open a separate LPS ticket for this.

### CODE REVIEW

**Comment by SelenaAungst** [comment]
ci:test:relevant

**Comment by SelenaAungst** [comment]
Hi @marcelabc @rebsilva is there anything else needed for this PR to pass review? Thank you!

**Comment by rebsilva** [comment]
Hi @SelenaAungst and @marcelabc!
I was on vacations and only came back yesterday.
I will take a look at the PR now, thanks! :cherry_blossom:

**Comment by SelenaAungst** [comment]
Ah sorry about that @rebsilva, I hope you enjoyed your vacation!

**Review by rebsilva — CHANGES_REQUESTED** [review]
Hey @SelenaAungst! Great work, but could you please do some changes? Since I was on vacations a new work started involving the Select fields on Forms, based on this [bug](https://liferay.atlassian.net/browse/LPS-195265). So the changes you did on the files Select.es.js and VisibleSelectInput.es.js won't be necessary because they are being completely reformulated; could you please remove your changes for both files? Thanks!

**Comment by rebsilva** [comment]
No need to be sorry @SelenaAungst!! And yes and enjoyed them very much, thanks! :blush: :purple_heart:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js [hunk 1]
```diff
@@ -113,6 +113,8 @@ const DocumentLibrary = ({
 							id={`${name}inputFile`}
 							lang={editingLanguageId}
 							onClick={onSelectButtonClicked}
+							readonly="true"
+							tabindex="-1"
```
[the remark this answers is not in the document]
Comment (SelenaAungst) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js:117]: Thanks for bringing this up! I will add these changes now

**Comment by SelenaAungst** [comment]
Hi @rebsilva, I made the changes you requested. Please let me know if there's anything else I can do for this!

**Review by rebsilva — APPROVED** [review]
LGTM! :tada: Good job @SelenaAungst! :cherry_blossom:

**Comment by rebsilva** [comment]
ci:test:sf

**Comment by rebsilva** [comment]
Some tests failing seem related so I'm taking a look now

**Comment by SelenaAungst** [comment]
Hi @rebsilva,
There was a change in Select.es.js.snap that I did not include because I removed all the changes to Select and VisibleSelectInput, should I add that back?

**Comment by rebsilva** [comment]
Hi @SelenaAungst! I think its better to delete all the test files for the Select, since this component won't exist anymore in the near future (a story is on the workings). I was thinking to take a look at these tests failing myself, what do you think? If you prefer to do the fixes let me know!

**Comment by SelenaAungst** [comment]
Hi @rebsilva,
I am not sure on this one, if they are going to delete Select.es.js in the future maybe they will want to delete the test files at the same time? I have not run into this issue before so whatever you think is best here!

**Comment by SelenaAungst** [comment]
Sounds good @rebsilva! I deleted the select test file and will leave you to your investigation. Thanks so much for looking into this!

**Comment by rebsilva** [comment]
I opened another PR with the tests fixed, so I will be closing this one @SelenaAungst https://github.com/liferay-forms/liferay-portal/pull/2376

**Pull request closed without merging**